src/web/modules/layout/layout.njk

```diff
@@ -52,6 +52,9 @@
             <li class="app-subnav__section-item">
               <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/transactions/search">Find a transaction</a>
             </li>
+            <li class="app-subnav__section-item">
+              <a class="app-subnav__link govuk-link govuk-link--no-visited-state" href="/transactions/statistics">Statistics</a>
```

Comment:
As we still need to play to the story to include the `WHERE live` condition on Ledger's end it would be good to remove the link from the main page. This will make it easy to see stats for a gateway account but not yet for the platform (as we don't want people to think the sandbox data are live payments)